circle.yml

```diff
@@ -12,7 +12,7 @@ dependencies:
     - security find-identity -p codesigning
     - instruments -s devices
     - xcodebuild -showsdks
-    - gem install fastlane -NVv 2.26.1
+    - gem install fastlane -NVv 2.28.2
```

Comment:
I've upped the Fastlane version as it includes a fix we'll need:

> Handle dSYM uploading for HockeyApp (#8863) via Prata Catalin

Ref: https://github.com/fastlane/fastlane/releases/tag/2.28.0